### ISSUE TITLE
Loss starting at 40k and not converging

### ISSUE THREAD

**Issue by lucastononrodrigues** [facebookresearch/detr#336]
I don't know why it is doing this but I am training the bdd100k dataset from scratch and when I do it I get a loss starting around 40k and it never converges. The same has already happened with COCO before.

To do this I just cloned the repository and then added a custom dataset file (just to change where to get the bdd100k converted to coco) and changed the number of classes to an argument args.num_class in the build_model from detr.py.

The most bizarre thing is that I have already fine-tuned the model before with a similar code (I don't recall what I changed if I did change something). Using this same dataset BDD100k which has 10 classes, I deleted the checkpoint['model'].class_embed bias and weights in order to finetune it. I have changed strict=false while loading the model as well.

Some other time when I tried to do this the loss started at 60 and went down to 5.

Therefore I saved the model and changed my machine but didn't save the code. When I tried to continue the training with this same model the loss starts at 40k, no matter what, even for this pre-trained/finetuned model that is actually predicting alright.

By loading this same finetuned model using the following code I get very good predictions. (cf images)

```
model = torch.hub.load('facebookresearch/detr',
                       'detr_resnet50',
                       pretrained=False,
                       num_classes=num_classes)


checkpoint = torch.load(checkpoint_path,
                        map_location='cpu')

model.load_state_dict(checkpoint['model'],
                      strict=False)
```

![image](https://user-images.githubusercontent.com/55460646/108114539-b1b85980-7077-11eb-8369-4bb1fbfe8a0c.png)
![image](https://user-images.githubusercontent.com/55460646/108114602-c3016600-7077-11eb-9025-29ab6844116c.png)

The model I am removing the heads for fine-tuning is this one:
```
torch.hub.load_state_dict_from_url(
            url='https://dl.fbaipublicfiles.com/detr/detr-r50-e632da11.pth',
            map_location='cpu',
            check_hash=True)
```

So to synthesize: 
-When I tried to train from scratch with COCO and BDD100k I got 40k loss that wouldn't converge.
-When I tried to continue the fine-tuning of a previously finetuned model the loss went to 40k as well, even though the model is actually doing good predictions.

Has anyone else had similar problems?

ps: I can send the model if somebody wants to give it a try.
ps2: The loss increases because the bounding box losses are extremely high.
ps3: One thing I did was that I wasn't normalizing this time, so the loss went down to 25 instead of 40k, but it doesn't change anything from the fact that I have a good model that has a weird loss when continuing training
